.github/workflows/lte-integ-test-bazel.yml

```diff
@@ -13,13 +13,9 @@ name: LTE integ test bazel
 
 on:
   workflow_dispatch: null
-  workflow_run:
-    workflows:
-      - build-all
+  push:
```

Comment:
- `github.event_name == 'workflow_dispatch'` needs to be removed. This probably holds for all the other workflows as well.
- As this workflow is "your" special bazel workflow, I did not check, if there are dependencies on build-all completion.

Reply:
- I disagree with the first point, the `workflow_dispatch` triggers are unaffected.

Reply:
Your are right, mixed up the `workflow_run` and `workflow_dispatch`. However, this raises the question, if we need the dispatch trigger at all. At least I do not see any necessity for it, if we run on every push on master anyway. So we might want to remove it altogether as well.

Reply:
We use the `workflow_dispatch` extensively for testing and to see if errors on master are reproducible.